common/artifacts/000_wc_artifacts.txt

```diff
@@ -356,6 +356,31 @@ atiesh = {
 	picture = "GFX_inv_staff_medivh"
 	slot = weapon
 }
+bloodhoof_runespear = {
+	quality = 5
+	flags = { combat_weapon spear }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
```

Comment:
Paladins and character without class can't equip it. Is it ok?
```suggestion
		OR = {
			NOT = { has_class_trait_trigger = yes }
			is_physical_class_trigger = yes
			is_combined_class_trigger = yes
		}
```

Reply:
done